api/services/ocean/spark/schemas/oceanSparkApplicationSpec.yaml

```diff
@@ -1226,22 +1226,19 @@ components:
       #example: [ '4096m', '16g' ]
       type: string
     OOMRecoveryStrategy:
-      descrtiption: Memory allocation strategy that automatically increments the memory when a previous app failed with a OOM
+      description: Memory allocation strategy that automatically increments the memory when a previous app failed with a OOM
       properties:
         strategy:
-          required: true
```

Comment:
where is the "required: true" defined now? Shouldn't it be before properties?

Reply:
I added it in the `required` block at the end of the object